clvm/__init__.py

```diff
@@ -1,5 +1,6 @@
 from .SExp import SExp
 from .operators import (  # noqa
+    QUOTE_ATOM,
```

Comment:
exposing my python-ignorance here; Does this import, being in `__init__` automatically cause it to be imported in all python files in this directory? If not, I don't understand the purpose of this.

Reply:
No, this is so you can write:

`from clvm import QUOTE_ATOM`

in other modules that import clvm. It's just exposing that binding at the root of the module, instead of having to say:

`from clvm.operators import QUOTE_ATOM`